lib/twingly/livefeed/version.rb

```diff
@@ -0,0 +1,5 @@
+module Twingly
+  module LiveFeed
+    VERSION = "5.0.1"
```

Comment:
Is it possible for us to have different versions for the livefeed module and/or search module? How would we handle the module version vs. gem version in that case? I'm thinking we need to move this to a shared version constant for both modules.